docs/Contentful.md

```diff
@@ -0,0 +1,43 @@
+# Contentful
+
+## Adding a new Richtext embed
+
+Create a new content model object in Contentful, and update the RichText elements to support including this content as an embed. Don't publish your changes until the frontend has been updated.
+
+Preview the API response of what this embed outputs by using the Contentful Preview API URL.
+
+`https://graphql.contentful.com/content/v1/spaces/[space ID here]/explore?access_token=[preview API key here]`
```

Comment:
Had a go at this but wasn't quite sure how to construct a query to see a youtube embed - the url to get to graphql worked though